packages/web/webpack.config.js

```diff
@@ -37,26 +36,26 @@ module.exports = {
                             ],
                         ],
                         plugins: [
-                            "@babel/plugin-transform-runtime",
-                            "@babel/plugin-transform-modules-commonjs",
+                            '@babel/plugin-transform-runtime',
+                            '@babel/plugin-transform-modules-commonjs',
                         ],
                     },
                 },
             },
             {
                 test: /\.wasm$/,
-                type: "asset/inline",
-            }
+                type: 'asset/inline',
+            },
         ],
     },
     experiments: {
         asyncWebAssembly: true,
     },
     output: {
-        filename: "[name].min.js",
-        path: path.resolve(__dirname, "lib"),
-        library: "concordium",
-        libraryTarget: "umd",
+        filename: '[name].min.js',
+        path: path.resolve(__dirname, 'lib'),
+        library: 'concordiumSDK',
```

Comment:
I wonder if there is a naming convention for library names, e.g. should it be `concordiumsdk` instead?

Reply:
Looking into it a bit, I can't say I can find a standard. I did find that you can do:
library: ['concordium', 'sdk']
and it should be available on `window.concordium.sdk`.
(But that would only work if we had the wallet api on `window.concordium.wallet`, and they would not overwrite each other.